site/_data/postsDataHooks/authors.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {index, individual} = require('./utils');
+
+/**
+ * @param {AuthorsItem[]} authors
+ * @param {string} locale
+ * @return {AuthorsItem[]}
+ */
+const authorsIndex = (authors, locale) => {
```

Comment:
This exists purely to make the types more specific, right? Might be worth a note in this jsdoc comment.

Can you also make it a one-liner like below?